configuration/configuration_blackbox_test.go

```diff
@@ -12,8 +12,8 @@ import (
 	"time"
 
 	jwt "github.com/dgrijalva/jwt-go"
-	"github.com/fabric8-services/fabric8-auth/configuration"
-	"github.com/fabric8-services/fabric8-auth/resource"
+	"github.com/fabric8-services/fabric8-wit/configuration"
```

Comment:
The same problem as with configuration. Please bring the original Auth config test back :)